Models/Events/MessageEventModel.cs

```diff
@@ -12,10 +12,12 @@ public class MessageEventModel : EventModel
         public string Text { get; set; }
         
         [JsonProperty("ts")]
-        [JsonConverter(typeof(UnixDateTimeConverter))]
-        public DateTime TimeSent { get; set; }
+        public decimal SlackTimeSent { get; set; }
+        
+        [JsonProperty("thread_ts")]
+        public decimal SlackThreadID { get; set; }
```

Comment:
`SlackThreadID` -> `SlackThreadId`

Do we need the additional precision that comes with `decimal`? Is `double` good enough?

Reply:
As discussed offline, we need the precision provided by decimal